tensorflow/lite/delegates/nnapi/java/src/main/java/org/tensorflow/lite/nnapi/NnApiDelegate.java

```diff
@@ -118,12 +118,24 @@ public Options setUseNnapiCpu(boolean enable) {
       return this;
     }
 
+    /**
+     * Enable or disable to allow fp32 computation to be run in fp16 in NNAPI.
+     * See https://source.android.com/devices/neural-networks#android-9
+     *
+     * <p>Only effective on Android 9 (API level 28) and above.
+     */
+    public Options setAllowFp16(boolean enable) {
```

Comment:
Can you also add a @Deprecated annotation to Interpreter.Options.setAllowFp16PrecisionForFP32 and point them to this?

Reply:
@freedomtan can you please add a @depricated

Reply:
@rthadur this is the newly added one. I added `@deprecated` to `tensorflow/lite/java/src/main/java/org/tensorflow/lite/Interpreter.java` as suggested by @jdduke. 

Original: set allow_fp16 option is associated with the Interpreter. This is considered deprecated.
change: an option added to the NNAPI delegate code, this is not deprecated.

> @freedomtan can you please add a @depricated